redfish-core/lib/log_services.hpp

```diff
@@ -1524,6 +1524,13 @@ inline void requestRoutesSystemLogServiceCollection(App& app)
         asyncResp->res.jsonValue["Members@odata.count"] =
             logServiceArrayLocal.size();
 #endif // BMCWEB_ENABLE_HW_ISOLATION
+
+#ifdef BMCWEB_ENABLE_LINUX_AUDIT_EVENTS
+        nlohmann::json::object_t auditLog;
+        auditLog["@odata.id"] =
+            "/redfish/v1/Systems/system/LogServices/AuditLog";
+        logServiceArray.push_back(std::move(auditLog));
```

Comment:
All of the other members added here are also using push_back(). Since this is just the odata.id and not the actual log entries I don't think the performance impact would be very much to change it.
I will leave it to match the others here.